internal/project/framework/internals/middleware/method_not_allowed_route_not_found.go

```diff
@@ -1,12 +1,13 @@
-{{ .Copyright }}
+/**#bean*/ /*#bean.replace({{ .Copyright }})**/
 package middleware
 
 import (
 	"net/http"
 	"sort"
 	"strings"
 
-	ierror "{{ .PkgPath }}/framework/internals/error"
+	/**#bean*/
+	ierror "demo/framework/internals/error" /*#bean.replace(ierror "{{ .PkgPath }}/framework/internals/error")**/
 
```

Comment:
put these two lines in one line

Reply:
`go fmt` command make it, `go fmt` will run before I commit,but I don't know the fmt rules
bean macro still works in two lines